### ISSUE TITLE
The training loop stops running when it has to create a new experiment see stdout below

### ISSUE THREAD

**Issue by bentherien** [sgaut023/kymatio_mod#4]
Train Epoch: 83  Average Loss: 0.863570, Accuracy: 73/100 (73.00%): 
Train Epoch: 84  Average Loss: 0.866516, Accuracy: 71/100 (71.00%): 
Train Epoch: 85  Average Loss: 0.866348, Accuracy: 73/100 (73.00%): 
Train Epoch: 86  Average Loss: 0.818349, Accuracy: 73/100 (73.00%): 
Train Epoch: 87  Average Loss: 0.936716, Accuracy: 64/100 (64.00%): 
Train Epoch: 88  Average Loss: 0.830300, Accuracy: 75/100 (75.00%): 
Train Epoch: 89  Average Loss: 0.947707, Accuracy: 66/100 (66.00%): 
Train Epoch: 90  Average Loss: 0.708818, Accuracy: 76/100 (76.00%): 
Train Epoch: 91  Average Loss: 0.897256, Accuracy: 74/100 (74.00%): 
Train Epoch: 92  Average Loss: 0.804800, Accuracy: 76/100 (76.00%): 
Train Epoch: 93  Average Loss: 0.866699, Accuracy: 72/100 (72.00%): 
Train Epoch: 94  Average Loss: 0.823398, Accuracy: 69/100 (69.00%): 
Train Epoch: 95  Average Loss: 0.904400, Accuracy: 76/100 (76.00%): 
Train Epoch: 96  Average Loss: 0.850978, Accuracy: 68/100 (68.00%): 
Train Epoch: 97  Average Loss: 0.812589, Accuracy: 76/100 (76.00%): 
Train Epoch: 98  Average Loss: 0.848335, Accuracy: 73/100 (73.00%): 
Train Epoch: 99  Average Loss: 0.951405, Accuracy: 70/100 (70.00%): 
Train Epoch: 100         Average Loss: 0.822586, Accuracy: 67/100 (67.00%): 

Test set: Average loss: 2.4348, Accuracy: 3388/10000 (33.88%)

INFO: 'linear_layer_baseline_v2' does not exist. Creating a new experiment
(torch11) benjamin@ben:~/Documents/github/kymatio_mod$
